### ISSUE TITLE
remove addresses for old street names

### ISSUE THREAD

**Issue by darrell** [pdxosgeo/pdxbldgimport#25]
Right now we're getting double address points on Rosa Parks (nee Portland Blvd) and Cesar Chavez (nee 39th Ave). We should delete the points for the old addresses. Are there other pairs we should be aware of?

**Comment by ksmpdx** [comment]
Wanted to let you know, per an earlier post, that the City of Portland has been refining it's address point data. If you can hold off on import until the next RLIS release in Feb, some of this stuff will be fixed, and the quality of the data will be a lot better.

**Comment by darrell** [comment]
Yes, we've asked for a sneak-peak of the data so we can integrate it, but haven't heard back.  I'm going to suggest that we hold off on doing Portland proper until after that release, or the city lets us have at the data.

**Comment by ksmpdx** [comment]
Sounds like a good plan. Still working on cleaning it up, hoping to get it most of the way there by the end of next week when we will deliver to Metro. Assume you all are getting the regional address points from them?

**Comment by darrell** [comment]
Yep, it all comes from RLIS right now.
